uplink/src/collector/logcat.rs

```diff
@@ -273,3 +271,12 @@ impl Logcat {
         });
     }
 }
+
+fn read_line_lossy(reader: &mut BufReader<ChildStdout>) -> Option<String> {
+    let mut buf = Vec::with_capacity(256);
+    if let Err(_) = reader.read_until(u8::try_from('\n').unwrap(), &mut buf) {
```

Comment:
1. Change to ` reader.read_until(b'\n', &mut buf)`?
2. Change `Option` to `Result` and break with error log